rllib/utils/typing.py

```diff
@@ -143,5 +144,26 @@
 # (possibly nested) dict|tuple of gym.space.Spaces.
 SpaceStruct = Union[gym.spaces.Space, dict, tuple]
 
+# A batch of RNN states with dimensions [state_index, batch, state_object].
+StateBatch = List[List[Any]]
```

Comment:
Can we call this `StateBatches` instead?

"A list of batches of RNN states. Each item in this list has dimension [B, S] (S=state vector size)"

Reply:
done